scrowl-io/scrowl
# [sc-5660] - Chore/Create main content panel

### Short summary
Create the main content panel and add the top cards, files table, and bottom cards.

1. Babel configs and typescript packages were added to fix Jest issues
2. The Electron Debug require statement was replaced to fix TS issues
3. Linting scripts ran in the whole project so you might notice some files with slight changes in the Files Changed list

---

### Test steps

1. Run `yarn start:tool:course` and ensure all the components were added to the main content panel
2. Although no new testes were added, run `yarn test:tool:course` to make sure all the existing tests will pass

---

### Screenshot
![Screen Shot 2022-04-14 at 1 23 44 PM](https://user-images.githubusercontent.com/98419841/163441429-0e6f5b7e-77f1-4cfa-8b21-b2c7f2210d3e.png)
